src/schema/objects/metadata.yaml

```
@@ -2757,6 +2936,19 @@ Resolution:
     - type: object
       additionalProperties:
         type: string
+ResonantNucleus:
+  name: ResonantNucleus
+  display_name: Resonant Nucleus
+  description: |
+    The isotope of interest of an MR experiment (for example, `"1H"`, `"13C"`, `"31P"`).
+    For multi-nuclei experiments such as <sup>1</sup>H-[<sup>13</sup>C] MR,
+    an array can be used: `["1H", "13C"]`.
+    Corresponds to DICOM Tag 0018, 9100 `Resonant Nucleus`.
+  anyOf:
+    - type: string
+    - type: array
+      items:
+        type: string
```

Looking through the MRS standard, this field is always an array. I suspect more people would value consistency between the MRS header and the BIDS sidecar than the ability to drop a pair of brackets:

```suggestion
  type: array
  items:
    type: string
```

---

Good point!